msal/src/main/java/com/microsoft/identity/client/internal/controllers/LocalMSALController.java

```diff
@@ -316,7 +337,12 @@ public AcquireTokenResult acquireTokenSilent(
                     new LocalAuthenticationResult(
                             fullCacheRecord,
                             cacheRecords,
-                            SdkType.MSAL
+                            SdkType.MSAL,
+                            strategy.getAuthorizationHeader(
+                                    parameters.getAuthenticationScheme(),
+                                    new TokenResponse() {{
+                                        setAccessToken(fullCacheRecord.getAccessToken().getSecret());
+                                    }}) // TODO this is a workaround
```

Comment:
TODO here, i see a fake token response is created, may be add an api with takes in just the AT and  Auth scheme and return the `authorization header` ?

Reply:
Resolved in 36b08f67f5b0eafe99733a7cd3031728888e0640